### PR TITLE
Project configuration updates for sdk 31

### PR DESCRIPTION
**Pull request by AdamKobus**

### CODE REVIEW

File: android/src/main/java/com/polidea/reactnativeble/utils/Base64Converter.java [hunk 1]
```diff
@@ -1,12 +0,0 @@
-package com.polidea.reactnativeble.utils;
```
Comment (AdamKobus) [inline android/src/main/java/com/polidea/reactnativeble/utils/Base64Converter.java:1]: unused

**Review by gegitor — DISMISSED** [review]

**Review by mciechanowicz — APPROVED** [review]

**Comment by anunes9** [comment]
@mciechanowicz any update when this pull request is gonna be merged?

**Review by mciechanowicz — APPROVED** [review]

**Comment by ankeknicker** [comment]
any update on this? would need it urgently

File: android/build.gradle [hunk 2]
```diff
@@ -3,48 +3,55 @@ def safeExtGet(prop, fallback) {
...
-        targetSdkVersion safeExtGet('targetSdkVersion', 27)
+        targetSdkVersion safeExtGet('targetSdkVersion', 31)
     }
     lintOptions {
         abortOnError false
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
```
Comment (sdandois) [inline android/build.gradle:34]: Java 8, still? This is no longer required by react-native

https://react-native-community.github.io/upgrade-helper/?from=0.61.0&to=0.67.3

**Comment by ericschaal** [comment]
Hey all! What's the status on this? :)

**Comment by jaska120** [comment]
Hi, how is this PR going? It would be very valuable to get this merged to allow SDK 31.

**Pull request closed without merging**